src/clr-angular/forms/styles/_textarea.clarity.scss

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@include exports('forms.control.textarea') {
+    .clr-textarea-wrapper {
+        white-space: nowrap;
+        display: flex;
+        margin-top: $clr-forms-baseline;
+        
+        .clr-validate-icon {
+            margin-left: 0;
+        }
+    }
+    
+    .clr-textarea {
```

Comment:
This pattern looks good to me. Nesting only when its needed.

Reply:
👍